src/app/layout.tsx

```diff
@@ -53,11 +55,13 @@ export default function RootLayout({
           gtag('config', '${NEXT_PUBLIC_GOOGLE_TAG_MANAGER_API_KEY}');
         `}
       </Script>
+
       <body>
         <CookiesProvider>
           <LanguageTranslationProvider>{children}</LanguageTranslationProvider>
         </CookiesProvider>
         <GoogleAnalytics gaId={GOOGLE_ANALYTICS_MEASUREMENT_ID} />
+        <GoogleTagManager gtmId="GTM-ND2QBSQH" />
```

Comment:
We have been able to send custom `geolocation` events:

https://github.com/streetlives/yourpeer.nyc/blob/9ec3566ecffb48360a0fde33eff18b1ad4ebbb9b/src/components/geo-context.tsx#L69

Is there a reason why we need a new tag and GoogleTagManager component?